CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Tests now wait for the `getRuntimeContext` request to finish before starting.
+
+### Fixed
+
+- Pickup + Delivery scenario in vtexgame1invoice account.
```

Comment:
@marcoskwkm it looks like the fix only applies to this account, correct?

Reply:
Yes, that's correct. The test was only failing in that account.

Reply:
No risk of failure on production accounts?

Reply:
I mean, the test failed only in this account because it requires invoice address, and the test was not doing the necessary extra steps in this case.